talent/job-descriptions/applications.md

```diff
@@ -1,6 +1,6 @@
 Engineer in the Applications team
```

Comment:
I'd replace the "Engineer" with "Software Engineer", even though by default it's a software engineering role. I think it's more concise.

Reply:
+1, I changed it